ui/src/views/image/RegisterOrUploadTemplate.vue

```diff
@@ -16,15 +16,16 @@
 // under the License.
 
 <template>
-  <div class="form-layout">
+  <div
+    class="form-layout"
+    @keyup.ctrl.enter="handleSubmit">
     <span v-if="uploadPercentage > 0">
       <a-icon type="loading" />
       {{ $t('message.upload.file.processing') }}
       <a-progress :percent="uploadPercentage" />
     </span>
     <a-spin :spinning="loading" v-else>
       <a-form
-        v-ctrl-enter="handleSubmit"
```

Comment:
@DaanHoogland Should we remove here as well ?

https://github.com/apache/cloudstack/blob/1a47cc3c954ded8db7afdc8a0154929ae54c6c48/ui/src/views/image/RegisterOrUploadIso.vue#L29

Reply:
yes, this is the one obfuscating the `space` event. that's why it was moved to the `div` element

Reply:
ok, is the same change required in _RegisterOrUploadIso.vue_

Reply:
I am working on that, but the RegisterOrUploadIso doesn't use `a-check`s but `a-switch`s, which work differently. these will not accept the  `space` event. I think we should merge like this and handle the rest when we encounter it.